wgpu/_classes.py

```diff
@@ -391,7 +391,7 @@ def info(self):
     def summary(self):
         """A one-line summary of the info of this adapter (description, adapter_type, backend_type)."""
```

Comment:
also device here

Reply:
and here https://github.com/pygfx/wgpu-py/blob/de55ef8eabd3359af24bbd9c77009eebe03e6c30/wgpu/backends/wgpu_native/_api.py#L348

Reply:
Nice catch, thanks!